### DISCUSSION TITLE
How can I disable resizing of tabsets?

### DISCUSSION THREAD

**Discussion by g-wm** [caplin/FlexLayout#461] [General]
The only method I have found is setting `splitterSize: 0` - there must be another way?

**Comment by Einar-san** [comment]
In my case I use onAction and a boolean state for it, you can do better : 
 `  function onAction(action: Action) {
      if (action.type === 'FlexLayout_AdjustSplit' && !isTabResizable) {
        return undefined
      }
}`

**Comment by powerdragonfire** [comment]
See [Issue #147](https://github.com/caplin/FlexLayout/issues/148#comment-composer-heading). Got working by using attributes maxWidth and minWidth. Also works when resizing parent window😊
